tensorboard/webapp/notification_center/_views/view_types.ts

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+/**
+ * A notification displayed in notification center.
+ */
+export type Notification = {
+  date: string;
+  title: string;
+  content: string;
+};
```

Comment:
Any reason why this is declared again?

Reply:
+1, if there is no view-specific data on this type, it would be convenient to drop this type and instead reuse the `Notification` type from `tensorboard/webapp/notification_center/types.ts`.

Reply:
This is kind of conflict with stephan's comment on `tensorboard/webapp/notification_center/_redux/BUILD` 
I think I want to have the type definition on `notification_center/type.ts` since this would be used in both `_view` and `_redux`
but then I need to create the deps in the _redux/BUILD  and _views/BUILD

Reply:
Sure, if having the `Notification` in `notication_center/type.ts` is convenient, sgtm.